data/mods/Magiclysm/monsters/zombified_monsters.json

```
@@ -32,5 +32,38 @@
     "death_function": [ "NORMAL" ],
     "upgrades": { "half_life": 28, "into_group": "GROUP_ZOLF_UPGRADE" },
     "flags": [ "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "KEENNOSE", "BASHES", "POISON", "NO_BREATHE", "REVIVES", "FILTHY" ]
+  },
+  {
+    "id": "mon_zorc",
+    "type": "MONSTER",
+    "name": { "str": "undead orc" },
+    "description": " This unholy amalgamation looks at you with a piercing stare so strong it might scare your soul away.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "categories": [ "CLASSIC" ],
+    "species": [ "ZOMBIE" ],
+    "volume": "67 L",
+    "weight": "200 kg",
+    "hp": 260,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "O",
+    "color": "yellow",
+    "aggression": 100,
+    "morale": 120,
+    "melee_skill": 7,
+    "melee_dice": 5,
+    "melee_dice_sides": 6,
+    "melee_cut": 6,
+    "dodge": 12,
```

```suggestion
    "dodge": 3,
```
I don't see any reason they'd be better at dodging than the blood orc.